snapcraft.yaml

```diff
@@ -0,0 +1,53 @@
+name: check-sieve
+license: MIT
+version: "0.6"
```

Comment:
The latest release is 0.8... or I guess it doesn't matter if it gets overridden below

Reply:
Ah, good catch. That version number is a typo.

Reply:
Bumped to `"0.8"`.